src/api/spec/controllers/webui/workflow_runs_controller_spec.rb

```diff
@@ -18,17 +18,51 @@
 
     context 'when action_filter is available' do
       it 'finds workflow runs when the action is available' do
-        get :index, params: { token_id: workflow_token.id, request_action: 'opened', generic_event_type: 'pull_request' }
+        get :index, params: { token_id: workflow_token.id, request_action: 'opened', pull_merge_request: 1 }
 
         expect(assigns(:workflow_runs).count).to eq(1)
```

Comment:
What's the difference to the context above? A count of `1` sound like a magic number to me... How about expecting the data your really expect in both?

Reply:
Fixed